packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -48,18 +48,18 @@ export const handler = Sentry.AWSLambda.wrapHandler(
         `dateTo: ${dateTo}, fileName: ${fhirFileName}`
     );
 
-    const bundle = await getBundleFromS3(fhirFileName);
-    log(`Bundle: ${JSON.stringify(bundle)}`);
-
     try {
       const log = prefixedLog(`patient ${patientId}`);
 
-      if (isSandbox) return processSandbox({ firstName, conversionType });
+      if (isSandbox) return processSandbox({ firstName, conversionType, bucketName });
 
       if (!converterKeysTableName) {
         throw new Error(`Programming error - SIDECHAIN_FHIR_CONVERTER_KEYS_TABLE_NAME is not set`);
       }
 
+      const bundle = await getBundleFromS3(fhirFileName, bucketName);
+      log(`Bundle: ${JSON.stringify(bundle)}`);
```

Comment:
Do we need this? Could just use the file name from the log and get it from S3? These bundles are huge and we can't do much with it on  CW logs

Reply:
I had just moved it but yea i agree this can be removed